habitat_sim/utils/viz_utils.py

```diff
@@ -94,3 +353,83 @@ def display_video(video_file: str, height: int = 400):
         else:
             opener = "open" if sys.platform == "darwin" else "xdg-open"
             subprocess.call([opener, video_file])
+
+
+def place_agent(sim, pos=[0.0, 0.0, 0.0], rot=np.quaternion(-1, 0, 0, 0)):
+    """    
+    Places our agent in the scene using specified position and orientation parameters
+
+    Parameters
+    ----------
+    sim : Simulator, 
+        Reference to the habitat simulator object
+    pos : List, optional
+        Location in scene to place agent. The default is [0.0,0.0,0.0].
+    rot : numpy quaternion, optional
+        Orientation of agent in scene.  The default is np.quaternion(-1, 0, 0, 0)
+
+    Returns
+    -------
+    agent's scene_node transformation matrix
+
+    """
+    agent_state = habitat_sim.AgentState()
+    agent_state.position = pos
+    agent_state.rotation = rot
+    agent = sim.initialize_agent(0, agent_state)
+    return agent.scene_node.transformation_matrix()
+
+
+def remove_all_objects(sim):
+    """
+    Removes all objects from simulation world
+
+    Parameters
+    ----------
+    sim : Simulator, 
+        Reference to the habitat simulator object
+
+    Returns
+    -------
+    None.
+
+    """
+
+    for id in sim.get_existing_object_ids():
+        sim.remove_object(id)
+
+
+def simulate(sim, dt=1.0, get_frames=True):
```

Comment:
Function name should be more descriptive, simulate_physics at the very least or visualize_physics?

Reply:
I tried to keep naming the same as the methods that have been used in existing tutorials, so that migration could occur more easily.  Of course they can be renamed, just the reason why they haven't been :).

Reply:
For all the tutorials that have been committed, you can easily refactor and test.